plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
I think our policy has been to just enable flake8 by default. All other checks are optional.